.cursor/hooks/post-edit-checks.sh

```
@@ -0,0 +1,121 @@
+}
+
+# Check 2: ESLint on changed files only
+eslint_files=""
+while IFS= read -r f; do
+  if [[ -f "$f" ]]; then
+    eslint_files="${eslint_files} ${f}"
+  fi
+done <<< "$changed_files"
+
+if [[ -n "$eslint_files" ]]; then
+  eslint_output=$(yarn eslint --no-error-on-unmatched-pattern --report-unused-disable-directives --max-warnings 0 $eslint_files 2>&1) || {
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)